src/lib/validations/sql.js

```diff
@@ -227,7 +227,7 @@ const checks: Checks = {
 		recommendation: '',
 	},
 	engineInnoDB: {
-		matcher: /ENGINE=(?!(InnoDB))/i,
+		matcher: / ENGINE=(?!(InnoDB))/i,
```

Comment:
Not sure I got this right, but if the leading space may or may not be there, what about

```js
/\s?ENGINE=(?!(InnoDB))/i
```

Reply:
That still flags it as an error with the example given in the issue. This isn't perfect and of course can be skipped with the `--skip-validate` option, but will alleviate the current issue.

Reply:
Got it. Thanks for clarifying!